modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -4,15 +4,28 @@
 	--n-accordion-border-width: 1px;
 	--n-accordion-border-color: #{$gray-light};
 	--n-accordion-border-style: solid;
+  	--n-accordion-item-title-space-between: 0px;
+  	--n-accordion-item-title-distance-from-content: 0px;
+  	--n-accordion-padding-top:0px;
+  	--n-accordion-padding-right:0px;
+  	--n-accordion-padding-bottom:0px;
+  	--n-accordion-padding-left:0px;
 
 	.e-n-accordion {
 
 		&-item {
 
+		  padding: var(--n-accordion-padding-top) var(--n-accordion-padding-right) var(--n-accordion-padding-bottom) var(--n-accordion-padding-left);
+
+		  &:not(:last-child) {
+				margin-bottom: var(--n-accordion-item-title-space-between);
```

Comment:
This might not be the best approach, as the .e-con element has margin settings that could overwrite this value. It me safer to add padding/margin/gap on the wrapper element...